components/style/binding_tools/regen.py

```diff
@@ -106,6 +106,7 @@
             "AnonymousContent",
             "AudioContext",
             "CapturingContentInfo",
+            "ConsumeStyleBehavior",
```

Comment:
For some reason without adding these namespace-less names here, the types didn't get output.